packages/data-explorer/src/routes.ts

```diff
@@ -1,26 +1,36 @@
 import MainView from './views/MainView.vue'
+import DataRowEdit from '@/components/DataRowEdit.vue'
 
-const routes = [
+export const defaultRouteQuery:any = { page: 1, size: 20, hideSidebar: 'false', view: 'CardView' }
+
+export const routes = [
   {
-    path: '/:entity/:view',
+    path: '/explorer/:entity',
     component: MainView,
-    name: 'main-view'
+    name: 'de-view'
+  },
+  {
+    path: '/explorer/:entity/create',
```

Comment:
Did you not say that you wont change the urls in this commit?

Reply:
Yes, that was to keep the link in the navigator working. I fixed the link in the navigator now instead.